ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/TutorialPageFragment.kt

```diff
@@ -50,7 +52,25 @@ internal class TutorialPageFragment() : Fragment(), TutorialCallbacks {
 
     private fun ViewDataBinding.setupCompose() {
         when (this) {
-            is TutorialTipsLearnBinding -> compose.setContent { TipsTutorialLayout() }
+            is TutorialTipsLearnBinding -> compose.setContent {
+                TipsTutorialLayout(
+                    R.raw.anim_tutorial_tips_people,
+                    R.string.tutorial_tips_learn_headline, R.string.tutorial_tips_learn_text, 0
```

Comment:
because the last parameter is only used for one of the pages, we can actually add a default value for it and then you won't have to specify `0` for the screens that don't have it